platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationFilesPoller.java

```diff
@@ -94,6 +97,7 @@ public void init() throws IOException {
     public void register(@NotNull ChangeListener listener) {
         notNull(listener, "ChangeListener cannot be null");
         changeListener = listener;
+        processExistingConfigurationFiles();
```

Comment:
This is better than it is currently for 2.10.  We should mark the fix for 2.10.1.